tests/partials/confirmation.spec.js

```diff
@@ -37,4 +37,15 @@ describe('confirmation template', () => {
 
 		expect($('dl dt').length).to.equal(2);
 	});
+
+
+	it('should display redirect to MMA', () => {
+		const details = [];
+		const $ = context.template({
+			details
+		});
+
+		expect($('dl dt').length).to.equal(0);
+		expect($.text()).to.contain('You can view details on your');
```

Comment:
I would suggest the following as it wont tie the test to the copy which will always change.
```suggestion
		expect($('a').length).to.equal(1);
```

Reply:
there are 3 'a' elements. I have added a filter, which would check for MMA link

Reply:
Not sure about the filter as the link may also change going forward (when we rebuild mma). You could use the `data-trackable="yourAccount"` though as that should never change.

Reply:
done ;)